pkgs/by-name/gq/gqmqtt/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "gqmqtt";
+  version = "0.2.0-alpha";
+
+  src = fetchFromGitHub {
+    owner = "klumw";
+    repo = "gqmqtt";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-4FV2Z3eow69v/Z6sfVfJew/N8ceiXX+JtvLidmiysPk=";
+  };
+
+  vendorHash = "sha256-4kT3dswD+Zlgal/kt3jOclDKkrBNXOZqvSPXg79TqX0=";
+
```

```suggestion

  passthru = {
    updateScript = nix-update-script { };
  };

```

You may want to add an update script to help with maintenance